docs/source/getting_started.md

```diff
@@ -0,0 +1,371 @@
+<!--
+SPDX-FileCopyrightText: Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+SPDX-License-Identifier: Apache-2.0
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Getting Started with Morpheus
+
+There are three ways to get started with Morpheus:
+- [Using pre-built Docker containers](#using-pre-built-docker-containers)
+- [Building the Morpheus Docker container](#building-the-morpheus-container)
+- [Building Morpheus from source](./developer_guide/contributing.md#building-from-source)
+
+The [pre-built Docker containers](#using-pre-built-docker-containers) are the easiest way to get started with the latest release of Morpheus. Released versions of Morpheus containers can be found on [NGC](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/morpheus/collections/morpheus_).
+
+More advanced users, or those who are interested in using the latest pre-release features, will need to [build the Morpheus container](#building-the-morpheus-container) or [build from source](./developer_guide/contributing.md#building-from-source).
+
+## Requirements
+- Pascal architecture GPU or better
+- NVIDIA driver `450.80.02` or higher
+- [Docker](https://docs.docker.com/get-docker/)
+- [The NVIDIA Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker)
+- [NVIDIA Triton Inference Server](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tritonserver) `22.06` or higher
+
+#### Note about Docker:
```

Comment:
`Note about Docker` is showing up as item in `Requirements` tree in left pane navigation.
<img width="309" alt="Screen Shot 2023-01-27 at 6 28 42 PM" src="https://user-images.githubusercontent.com/48110516/215225275-2f294319-cab1-4735-88eb-d24fae569327.png">

Reply:
Good catch